api/controllers/game/play-again.js

```diff
@@ -0,0 +1,128 @@
+      const newGame = await gameAPI.createGame(game.name, shouldNewGameBeRanked);
+
+      // Switch pNums
+
+      const newP0 = await User.findOne({ id: updatedGame.p1 });
+      const newP1 = await User.findOne({ id: updatedGame.p0 });
+      await Game.replaceCollection(newGame.id, 'players').members([newP0.id, newP1.id]);
+
+      Game.publish([game.id], {
+        verb: 'updated',
+        change: 'newGameForRematch',
+        game: updatedGame,
+        pNum,
+        gameId: newGame.id,
+      });
+
+      await User.updateOne({ id: newP0.id }).set({ pNum: 0 });
+      await User.updateOne({ id: newP1.id }).set({ pNum: 1 });
+
+      // Create Cards
+      return new Promise(function makeDeck(resolveMakeDeck) {
+        const game = newGame;
+        const findP0 = userService.findUser({ userId: newP0.id });
+        const findP1 = userService.findUser({ userId: newP1.id });
+        const data = [Promise.resolve(game), findP0, findP1];
+        for (let suit = 0; suit < 4; suit++) {
+          for (let rank = 1; rank < 14; rank++) {
+            const promiseCard = cardService.createCard({
+              gameId: game.id,
+              suit,
+              rank,
+            });
+            data.push(promiseCard);
+          }
+        }
+        return resolveMakeDeck(Promise.all(data));
+      })
+        .then(function deal(values) {
+          const [game, p0, p1, ...deck] = values;
+
+          // Shuffle deck & map cards => thier ids
+          const shuffledDeck = _.shuffle(deck).map((card) => card.id);
+          // Take 1st 5 cards for p0
+          const dealToP0 = shuffledDeck.splice(0, 5);
+          // Take next 6 cards for p1
+          const dealToP1 = shuffledDeck.splice(0, 6);
+          // Take next 2 cards for topcard & secondCard
+          gameUpdates.topCard = shuffledDeck.shift();
+          gameUpdates.secondCard = shuffledDeck.shift();
+          gameUpdates.lastEvent = {
+            change: 'Initialize',
+          };
+          gameUpdates.p0 = p0.id;
+          gameUpdates.p1 = p1.id;
+          gameUpdates.p0Ready = true;
+          gameUpdates.p1Ready = true;
+
+          // Update records
+          const updatePromises = [
+            // Deal to p0
+            User.replaceCollection(p0.id, 'hand').members(dealToP0),
+            // Deal to p1
+            User.replaceCollection(p1.id, 'hand').members(dealToP1),
+            // Replace Deck
+            Game.replaceCollection(game.id, 'deck').members(shuffledDeck),
+            // Other game updates
+            Game.updateOne({ id: game.id }).set(gameUpdates),
+          ];
+
+          return Promise.all([game, p0, p1, ...updatePromises]);
+        })
+        .then(function getPopulatedGame(values) {
+          return gameService.populateGame({ gameId: values[0].id });
+        })
+        .then(function publish(fullGame) {
+          Game.publish([fullGame.id], {
+            verb: 'updated',
+            change: 'Initialize',
```

Comment:
We've switched this to lowercase now
```suggestion
            change: 'initialize',
```